src/modules/todo.js

```diff
@@ -0,0 +1,61 @@
+import trash from '../trash.svg';
+import { getItem, setItem } from './localStorage.js';
+
+export default class Todo {
+  constructor() {
+    this.todoList1 = getItem() || [];
+  }
+
+  newTodolist = () => {
+    const formToReset = document.getElementById('frm');
+    const newInput = document.getElementById('new-todo');
+    newInput.addEventListener('keypress', (e) => {
+      if (e.code === 'Enter') {
+        if (e.target.value.length > 0) {
+          const value = {
+            description: e.target.value,
+            isCompleted: false,
+            index: this.todoList1.length,
+          };
+          this.todoList1.push(value);
+          setItem(this.todoList1);
+          formToReset.reset();
+        }
+      }
+      this.getList();
+    });
+  }
+
+  clearLocalStorage = () => {
+    const listToclear = this.todoList1.filter((todo) => !todo.isCompleted);
+    setItem(listToclear);
+    this.todoList1 = listToclear;
+    window.location.reload();
+  }
+
+  getList = () => {
+    const lst = document.getElementById('list');
+    lst.innerHTML = this.todoList1.map((el, index) => (` <li class="list">
```

Comment:
Hey bizip, great work implementing logic in a separate class !
what if there is a way to make the class more pure, what do you think if we write dom-manipulation away from that class?
you can also check [javascript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)